src/fontra/client/core/path-functions.js

```diff
@@ -626,11 +626,14 @@ export function scalePoint(pinPoint, point, factor) {
 export function getSelectionByContour(path, pointIndices) {
   const selectionByContour = new Map();
   for (const pointIndex of pointIndices) {
-    const [contourIndex, contourPointIndex] = path.getContourAndPointIndex(pointIndex);
-    if (!selectionByContour.has(contourIndex)) {
-      selectionByContour.set(contourIndex, []);
+    if (pointIndex !== undefined && pointIndex < path.numPoints) {
```

Comment:
I don't think `pointIndex` can be undefined.

But please rewrite in this form:

```js
    if (condition not met) {
      continue;
    }
```

Reply:
About `pointIndex` == 'undefined': I copied it from here:
https://github.com/googlefonts/fontra/blob/31cffdbd507d027c9724512c1f36b4d6aa21eb6d/src/fontra/views/editor/edit-tools-pen.js#L623

Reply:
I removed the `pointIndex !== undefined` + `continue` if `pointIndex >= path.numPoints`

Reply:
> I copied it from here

Why, though? That situation is not comparable at all. Please don't blindly copy/paste/copilot.